docs/list.md

```diff
@@ -94,6 +94,13 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Once all 4 PRs are accepted, then [fill out the form here](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 - **Notes**: Additional information regarding their Hacktoberfest participation [can be found on this page](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 
+#### **Chimoney**
```

Comment:
This entry is a bit light. Can you please try to add some additional detail? Like a link to a blog post about their Hacktoberfest plans? Tweets? Something to confirm their participation? Please go off some of the other entries on the Swag List for reference. Thank you!